pyproject.toml

```diff
@@ -0,0 +1,74 @@
+[build-system]
+requires = ["setuptools>=65"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "simtools"
+description = "Tools for the Simulation System of the CTA Observatory"
+readme = "README.rst"
+license = { file="LICENSE" }
+version = "0.4.0"
+authors = [
+  { name="Orel Gueta", email="orel.gueta@desy.de" },
+  { name="Victor Barbosa Martins", email="victor.barbosa.martins@desy.de" },
+  { name="Gernot Maier", email="gernot.maier@desy.de" },
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: BSD-3-Clause",
+    "Operating System :: OS Independent",
+]
+requires-python = ">=3.9"
+dependencies = [
+    "astropy",
+    "eventio",
+    "matplotlib",
+    "numpy",
+    "pyyaml",
+    "pymongo",
+    "pyproj",
+    "scipy",
+    "zstandard",
+]
+
+[project.optional-dependencies]
+"tests" = [
+    "pytest",
+    "pytest-cov",
+]
+"dev" = [
+    "pre-commit",
+    "pyflakes",
+]
+"doc" = [
+    "numpydoc",
+    "sphinx",
+    "sphinx_rtd_theme",
+]
+
+[project.urls]
+"documentation" = "https://gammasim.github.io/simtools/"
+"repository" = "https://github.com/gammasim/simtools"
+"bug tracker" = "https://github.com/gammasim/simtools/issues"
+
+[project.scripts]
+simtools-add-file-to-db = "simtools.applications.add_file_to_db:main"
+simtools-compare-cumulative-psf = "simtools.applications.compare_cumulative_psf:main"
+simtools-db-development-tools = "simtools.applications.db_development_tools:main"
```

Comment:
This is not an application

Reply:
Yes, it is a directory. I've removed it.